docs/examples/traefik-ingress.md

```diff
@@ -1,24 +1,25 @@
-# Installing the Traefik Ingress Controller on k0s
+# Install the Traefik Ingress Controller on k0s
 
-In this tutorial, you'll learn how to configure k0s with the
-[Traefik ingress controller](https://doc.traefik.io/traefik/providers/kubernetes-ingress/),
-a [MetalLB service loadbalancer](https://metallb.universe.tf/),
-and deploy the Traefik Dashboard along with a service example.
-Utilizing the extensible bootstrapping functionality with Helm, 
-it's as simple as adding the right extensions to the `k0s.yaml` file 
-when configuring your cluster.
+You can configure k0s with the [Traefik ingress
+controller](https://doc.traefik.io/traefik/providers/kubernetes-ingress/), a
+[MetalLB service loadbalancer](https://metallb.universe.tf/),
+and deploy the Traefik Dashboard using a service sample. With the extensible
+bootstrapping functionality that Helm provides, this is as simple as adding the
+several extensions to the `k0s.yaml` file  when you configure your cluster.
 
+## 1. Configure k0s.yaml
 
-## Configuring k0s.yaml
+Configure Traefik and MetalLB Helm charts to install during cluster bootstrap
+by modifing your k0s configuration file (`k0s.yaml`) to include the Traefik and
+MetalLB helm charts as extensions.
 
-Modify your `k0s.yaml` file to include the Traefik and MetalLB helm charts as extensions,
-and these will install during the cluster's bootstrap.
+##### Note
 
-> Note: You may want to have a small range of IP addresses that are addressable on your network,
-preferably outside the assignment pool allocated by your DHCP server.
-Providing an addressable range should allow you to access your LoadBalancer and Ingress services
-from anywhere on your local network.
-However, any valid IP range should work locally on your machine.
+A good practice is to have a small range of IP addresses for MetalLB on your
+LAN that are addressable, preferably outside the assignment pool your DHCP
```

Comment:
The intent in the original snippet is "addressable on your network", this seems to change that to "IP addresses that are addressable", which, I think, does not compute.

Reply:
Tweaked it back for the most part. The note now reads: 

"A good practice is to have a small range of IP addresses  that are addressable on your network, preferably outside the assignment pool your DHCP server allocates (though any valid IP range should work locally on your machine). Providing an addressable range allows you to access your LoadBalancer and Ingress services from anywhere on your local network."